config/Mainz-MogonNHR/config.yml

```
@@ -14,11 +14,18 @@ repo: "https://github.com/snakemake-workflows/transriptome-differential-expressi
 ## Workflow-specific Parameters:
 
 # the reference genome respectively transcriptome, is defined here:
+# You can either provide local reference data or an NCBI accession number do download from
 ref:
     species: "Chironomus riparius"
+    # Genome, allowed extensions:  .FA or .FASTA or .FNA
+    genome: "/lustre/project/nhr-zdvhpc/transcriptome_data/GCA_917627325.4_PGI_CHIRRI_v4_genomic.fa"
+    # Annotation, allowed extensions: .GFF or .GTF
+    annotation: "/lustre/project/nhr-zdvhpc/transcriptome_data/GCA_917627325.4_PGI_CHIRRI_v4_genomic.gff"    
```

yamllint

[error] trailing spaces

(trailing-spaces)